components/inputs/input-date-range.js

```diff
@@ -50,7 +53,9 @@ class InputDateRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 			/**
 			 * Value of the start date input
 			 */
-			startValue: { attribute: 'start-value', reflect: true, type: String }
+			startValue: { attribute: 'start-value', reflect: true, type: String },
+			_endLabel: { type: String },
+			_startLabel: { type: String }
```

Comment:
Probably want `attribute: false` here so that setting a `_endlabel` attribute on the element doesn't set this.